cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from cinema.views import movie_list, movie_detail
+
+app_name = "cinema"
+
+urlpatterns = [
+    path("movies/", movie_list, name="moive_list"),
```

Comment:
There is a typo in the URL name 'moive_list'. It should be 'movie_list' to match the view function and maintain consistency.